src/routes/index.js

```diff
@@ -168,7 +170,18 @@ router.get('/loginSuccess', async (req, res) => {
 
   await req.user.save();
 
-  res.redirect('/lobby');
+  // TODO-kev: Consider alternative areas to put check
+  // Update Patreon status
+  const patreonAgent = new PatreonAgent(new PatreonController());
+  const patronDetails = await patreonAgent.findOrUpdateExistingPatronDetails(
+    req.user.usernameLower,
+  );
+
+  if (patronDetails && !patronDetails.isPledgeActive) {
+    res.redirect('/lobby?patreonExpired=true');
```

Comment:
Let's shove it onto the user model as a single var for now. 
Future work to extend the notification system to allow for swals and stuff.